### DISCUSSION TITLE
Zoom Members

### DISCUSSION THREAD

**Discussion by ISchwandner** [IBAHCM/RPiR#414] [Old]
To share your github repositories for Practical 1-6.

**Comment by ISchwandner** [comment]
Imogen Schwandner, ISchwandner : https://github.com/IBAHCM/ISchwandner_series01.git

I forgot to reinstall RPiR today before completing all the steps, so I would be interested in seeing if that causes any issues. Thanks!

**Reply by maryadejoh** [reply]
Does not generate any issues here and everything looks fine. Cheers!

**Reply by ISchwandner** [reply]
Thank you!

**Comment by DylanCarbone** [comment]
My name's Dylan Carbone and my account is called DylanCarbone.
My git repository can be found here: https://github.com/IBAHCM/DylanCarbone_series01

Please let me know know if you have any comments or issues. Thanks!

**Comment by ctbautista88** [comment]
I am Criselda Bautista. Sharing my git repository here: https://github.com/IBAHCM/ctbautista88_series01 

Feel free to access the repository. :)

**Comment by AggieMatope** [comment]
Hi, my name is Agnes Matope. You will find my repository here: https://github.com/IBAHCM/agnesmatope_0201 

Will be happy to hear your feedback. Many thanks!

**Reply by maryadejoh** [reply]
Hi Agnes, I cannot access your repository. Kindly check to see if your repository can be cloned.
Regards.

**Reply by AggieMatope** [reply]
Hi Mary, I realised that I didn't give you access to my repository (my  apologies). You should have access now. 

Thanks
Agnes

**Reply by maryadejoh** [reply]
Got it now. Cheers

**Comment by sreedevi-gummuluri** [comment]
Hi There, I'm Sreedevi Gummuluri. My repository is located here: https://github.com/IBAHCM/sreedevi-gummuluri_series01.git.

Please let me know if you have any issues.

Thank you! 
Sree

**Reply by maryadejoh** [reply]
Repository accessed. No issues encountered

**Comment by maryadejoh** [comment]
Hello everyone. My name is Mary Adejoh and my repository can be found here:
https://github.com/IBAHCM/maryadejoh_series01.git

Kindly leave feedback and suggestions.
Thanks.

**Reply by sreedevi-gummuluri** [reply]
HI Mary, your repository worked well and I noted no issues. thanks!

**Reply by maryadejoh** [reply]
Thanks, Sree

**Comment by ISchwandner** [comment]
Hey Zoomers,

I have started a new discussion for us for practical series 2, since we are quite a lot of people and things might get lost otherwise.

Here is the link: #425
